typescript/formsApps.d.ts

```diff
@@ -45,6 +45,10 @@ export type FormsAppScheduledTasksMenuItem = FormsAppBaseMenuItem & {
   tasks: Array<{ taskId: number }>
   /** If true, menu item will be the default item shown */
   isDefault: boolean
+  /** The ids of the task groups assigned to the app */
+  taskGroups: Array<{ taskGroupIds: number }>
```

Comment:
```suggestion
  taskGroups: Array<{ taskGroupId: number, taskGroupInstanceId: string, label: string }>
```
eg an instance of Pumper (taskGroupId 1), instanceId: abc123 (guid), and label - 'Pumper 123'

Reply:
I think the API will probably need to assign the GUID so we can track task completion properly but it needs more thought (from me anyway)

Reply:
I wouldn't be against a separate screen to define the group instance or item or whatever we call it.  Then we can a menu item called "Reusable Tasks" or "Repeated Tasks" whatever makes sense to the user and then have menu item called "Items" and you assign items to the app, not task groups.  To me that's the cleanest way of doing this from an E2E perspective.